gql/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: gql
-version: 0.14.0
+version: 0.15.0
```

Comment:
Since we are still at version <1.0.0, a bump from 0.14.0 to 0.15.0 would indicate a breaking change like a major version bump according to 
[pub_semver](https://pub.dev/packages/pub_semver).

Since gql is used by [many packages](https://pub.dev/packages?q=dependency%3Agql), major version bumps like this cause a lot of pain for user, so I would like to avoid them when possible.

Since these changes are backwards compatible, I'd prefer to just bump to 0.14.1.  

I think it's time for many packages of the gql ecosystem to be moved to 1.0 version soon, and I hope I'll find time to do that soon, so discussion like this can be avoided in the future :)